CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 
 ## In Development
-
+* Drop Helm `v2` support and fully migrate to Helm `v3` (#163)
+* Switch dependencies from deprecated `helm/charts` to new Bitnami Subcharts (#163)
```

Comment:
@christopher-russell Feel free to also include your name or nickname here as someone who contributed this work.